Assets/HoloToolkit/Input/Scripts/Focus/InputSourcePointer.cs

```diff
@@ -34,6 +36,10 @@ public RayStep[] Rays
 
         public PointerResult Result { get; set; }
 
+        public GameObject Target { get; set; }
```

Comment:
Not convinced we need this here.  It should be enough that we pass this info through the raise event

Reply:
Read the section **IFocuser, IPointingSource & PointerResult** where I go over this change in detail.